tests/unittests/regression/test_pearson.py

```diff
@@ -84,13 +112,17 @@ def test_pearson_corrcoef(self, preds, target, compute_on_cpu, ddp):
             metric_args={"num_outputs": num_outputs, "compute_on_cpu": compute_on_cpu},
         )
 
-    def test_pearson_corrcoef_functional(self, preds, target):
+    def test_pearson_corrcoef_functional(self, preds, target, metric_args):
         """Test functional implementation of metric."""
         self.run_functional_metric_test(
-            preds=preds, target=target, metric_functional=pearson_corrcoef, reference_metric=_scipy_pearson
+            preds=preds,
+            target=target,
+            metric_functional=pearson_corrcoef,
+            reference_metric=partial(_ref_metric, metric_args=metric_args),
+            metric_args=metric_args,
```

Comment:
```suggestion
            **metric_args,
```
this needs to be provided in this way to be redirected to the update method correctly